subset/switches/src/main/java/switchtest/Main.java

```diff
@@ -4,18 +4,19 @@
 
   public static void main(String[] args) throws Exception {
 
-    if (args.length != 1) {
-      throw new IllegalArgumentException("Expected ipAddress && port as argument");
+    if (args.length != 3) {
+      throw new IllegalArgumentException("Expected ipAddress && port && supportPOE as arguments");
     }
 
     String ipAddress = args[0];
 
     int interfacePort = Integer.parseInt(args[1]);
 
-    SwitchInterrogator switchInterrogator = new SwitchInterrogator(ipAddress, interfacePort);
+    boolean supportsPOE = (args[2].equals("true")) ? true : false;
```

Comment:
Not sure you need a ternary here - `(args[2].equals("true"))` will return true or false anway, if I'm not mistaken?

Reply:
Other than that though, LGTM

Reply:
thanks, good spot, just waiting for travis to finish build then will update,